update-common/src/artifacts/update_plan.rs

```diff
@@ -666,41 +672,88 @@ impl<'a> UpdatePlanBuilder<'a> {
         artifact_id: ArtifactId,
         stream: impl Stream<Item = Result<bytes::Bytes, tough::error::Error>> + Send,
     ) -> Result<(), RepositoryError> {
-        if self.host_phase_1.is_some() || self.host_phase_2_hash.is_some() {
+        if self.cosmo_host_phase_1.is_some()
+            || self.gimlet_host_phase_1.is_some()
+            || self.host_phase_2_hash.is_some()
```

Comment:
Just for my own understanding. Host phase 2 artifacts will always be the same regardless of the type of sled, yes? Do you have a link to an RFD, or somewhere where I can read more on how this works?

Reply:
@karencfv that's correct: the Phase 2 image is common across sled types. Phase 1 differs only because we have different firmware blobs and things (from AMD) that we vary across microarchitectures (Milan vs Turin) that need to be loaded by the PSP; the x86 code (`phbl` and the OS bits) are identical, however.